### DISCUSSION TITLE
.NET 6 compatibility

### DISCUSSION THREAD

**Discussion by Marv51** [microsoft/WindowsAppSDK#1755] [General] [answered]
Please make a statement on if .NET 6 is officially supported for the published releases. The dotnet team is saying ".NET 6 is ready for your app." But they do not clarify if this also refers to WinUI/Windows App SDK Apps.

**Comment by harvinders** [comment]
It works, I have my app that targets .8 release with .Net 6.

**Comment by andrewleader** [comment, marked as answer]
Thanks for pointing this out @Marv51, Windows App SDK 1.0 is officially supporting .NET 6, we'll update our docs to make that clear!
